examples_tests/01.HelloWorld/main.cpp

```diff
@@ -107,14 +114,14 @@ int main()
 {
 	constexpr uint32_t WIN_W = 800u;
 	constexpr uint32_t WIN_H = 600u;
-	constexpr uint32_t SC_IMG_COUNT = 3u;
+	constexpr uint32_t SC_IMG_COUNT = 3u; // problematic, shouldn't fix the number of swapchain images at compile time, since Vulkan is under no obligation to return you the exact number of images you requested

```

Comment:
make it a variable, pass by reference to the swapchain creation functio, get the real count out

Reply:
Resolved